sql/moz-fx-data-shared-prod/snowflake_migration_derived/corpus_items_current_v1/view.sql

```diff
@@ -0,0 +1,26 @@
+CREATE OR REPLACE VIEW
+  `moz-fx-data-shared-prod.snowflake_migration_derived.corpus_items_current_v1`
+AS
+WITH corpus_item_id_row_nums AS (
+  SELECT
+    approved_corpus_item_external_id,
```

Comment:
```suggestion
    corpus_item_id,
```
This is the correct field name